template-sample-app/template/src/components/views/Kpi.js

```diff
@@ -14,10 +14,13 @@ import {
 import { AggregationTypes, CategoryWidget, FormulaWidget } from '@carto/react/widgets';
 
 import { currencyFormatter } from 'utils/formatter';
+import { kpiSource, KPI_SOURCE_COLUMNS } from 'data/sources/KpiSource';
 
 export default function Kpi() {
   const dispatch = useDispatch();
 
+  const LAYER_ID = 'kpiLayer';
```

Comment:
After the good refactor you have done on Source, removing the const here and exposing it from the 'source' file, I think we should probably do the same for layers: exporting a const with the 'id' from each Layer file, and consuming it here with kpiLayer.id. That way, there is just one place for the LAYER_ID, and that is the layer itself, which IMO it makes sense...

what do you think?

Reply:
It's being done in #167.